lib/pltr/v2/reducers/tours.js

```diff
@@ -1,5 +1,6 @@
-import { SET_TOUR_FEATURE, TOUR_NEXT, TOUR_RESTART } from '../constants/ActionTypes'
+import { SET_TOUR_FEATURE, TOUR_NEXT, TOUR_TRANSITION, TOUR_RESTART } from '../constants/ActionTypes'
 import { tour } from '../store/initialState'
+import { ACTS_TOUR_STEPS } from '../../../../src/app/components/intros/actsTourSteps'
```

Comment:
We can't have a library depend on the app that it's embedded into.  That's a circular dependency.
We need to find another way to do this.